lib/utils/icon_sets.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:flutter/material.dart';
+
+enum GraphicAssetType { childAvatars, awardsIcons, badgeIcons }
+
+class GraphicAsset {
+	final String filename;
+	final String label;
```

Comment:
I assume that will be used for filtering later? If so can we make it an enum?

Reply:
I know I know, we probably want a different enum for each category. Well, we could either extend GraphicAsset three times, or do that:
![image](https://user-images.githubusercontent.com/8643919/87952062-a4b79580-caa9-11ea-9e7b-fe9e4946bbcc.png)
Either way we need a cast to access the label value

Reply:
Hmm, but why splitting it? Can't we have just one GraphicLabel enum with labels? (If you have string so much 😆) They probably will never not overlap and even if - there will be no duplicates in one GraphicType.

`enum GraphicLabel  { boy, girl, gift, relax, activity (...) }`

Reply:
My point was that with enums we can:
- enumerate (daaa)
- get index
- get name
which simplifies things if we would like for example to show a combo box that filters the visible icons (and there are many). If we had one enum for all categories we wouldn't know which part of it to show in such ui element.

Reply:
Ok, didn't think of it as categories - just labels.. like on tooltips.
Changed applied.

Reply:
When labels start to repeat they become categories ;)